<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Connection</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>IP</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ip_line_edit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Port</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="port_spin_box">
            <property name="minimum">
             <number>1025</number>
            </property>
            <property name="maximum">
             <number>65000</number>
            </property>
            <property name="value">
             <number>5025</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="connect_button">
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="connection_status_line_edit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>connection status: disconnected</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>Screenshots</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QPushButton" name="save_screen_button">
            <property name="text">
             <string>Save screen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="screenshot_filename_line_edit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="placeholderText">
             <string>filename.ext</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QPushButton" name="set_color_theme_button">
            <property name="text">
             <string>Set color theme</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="theme_combo_box">
            <item>
             <property name="text">
              <string>Standart</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Monochrome</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Flat color</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Flat monochrome</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Freq settings</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_16">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Start Freq</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="start_freq_spin_box">
            <property name="maximum">
             <number>999999999</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="start_freq_units_combo_box">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>Hz</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>kHz</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>MHz</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>GHz</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>End Freq</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="stop_freq_spin_box">
            <property name="maximum">
             <number>999999999</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="stop_freq_units_combo_box">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>Hz</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>kHz</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>MHz</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>GHz</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="set_freq_button">
          <property name="text">
           <string>Set Frequency</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Center Freq</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="center_freq_spin_box">
            <property name="maximum">
             <number>999999999</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="center_freq_units_combo_box">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>Hz</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>kHz</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>MHz</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>GHz</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Span</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="span_freq_spin_box">
            <property name="maximum">
             <number>999999999</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="span_freq_units_combo_box">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>Hz</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>kHz</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>MHz</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>GHz</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="set_freq_span_button">
          <property name="text">
           <string>Set Frequency</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Peak measure settings</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Trig min, dBm</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="trig_min_spin_box">
            <property name="maximum">
             <number>30</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Trig max, dBm</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="trig_max_spin_box"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_17">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Sorting order</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="peak_order_combo_box">
            <item>
             <property name="text">
              <string>Amplitude</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Frequency</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Time</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="measure_peaks_button">
          <property name="text">
           <string>Measure Peaks</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Measure result</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>CH num</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="meas_ch_num_line_edit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Amplitude, dBm</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="meas_ampl_line_edit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Freq</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="meas_freq_line_edit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="power_down_button">
     <property name="text">
      <string>Power down N9010A</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
